tests/dns-eve-log-https-only/test.yaml

```diff
@@ -0,0 +1,9 @@
+requires:
+  min-version: 7
+checks:
```

Comment:
just a minor nit: for legibility, we should probably have a newline here.

Reply:
I added a line after requires block(new PR). But now I am confused. Were you referring to line 3?